static/src/stylesheets/layout/new-header/_veggie-burger.scss

```diff
@@ -58,6 +58,25 @@
     }
 }
 
+.veggie-burger__label {
+    @include fs-textSans(5);
+    bottom: -25px;
+    color: $neutral-3;
+    display: none;
```

Comment:
So this means that screen readers on mobile won't be able to read this label?

Before it used a class which used [this mixin](https://github.com/guardian/frontend/blob/0e2e33174311b9144dd170006d504b77b9663d61/static/src/stylesheets/_mixins.scss#L74) which made it possible